factory/mock/oneShardCoordinatorMock.go

```diff
@@ -0,0 +1,60 @@
+package mock
```

Comment:
Do we still need this? We can use multiShardCoordinatorMock.

Reply:
right. removed